### ISSUE TITLE
Most Beloved Book on Instance has zero local reviews

### ISSUE THREAD

**Issue by realityfabric** [bookwyrm-social/bookwyrm#1978]
**Describe the bug**
On the About this Instance page there's a cool section that shows various books, like Most Beloved, Most Contentious, Most Wanted to Read. The Most Beloved category, however, appears to be driven by federated statistics and not local statistics. A book with no local ratings is listed as the most Beloved Book because *other* instances have rated it highly. 

**To Reproduce**
1. Don't have a lot of ratings on your instance.
2. Import a book that is highly rated from another (in this case 3 reviews, all 5 stars) 
3. Go to instancedomain/about and view the Most Beloved book

I'm really only guessing on these steps to reproduce, it's just something I stumbled across.

**Expected behavior**
I expect the most beloved book to be beloved by my instance, not other instances. Federated statistics would also be cool, but it doesn't seem like this section is meant to represent the network. 

**Instance**
https://tankie.ml/

---

**Smartphone (please complete the following information):**
    - Device: Samsung Galaxy A52 5G
    - OS: Android 12
    - Browser: Firefox
    - Version: 97.2.0

**Comment by mouse-reeve** [comment]
Yep, I'm able to replicate this locally as well. I did write the query with the intention of it filtering to only local reviews, but I must be misunderstanding something about what I wrote. Thanks for letting me know!
